src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -694,7 +694,7 @@ private static RuntimeType ResolveType(RuntimeModule scope, string typeName)
 
             if (type == null)
                 throw new InvalidOperationException(
-                    string.Format(CultureInfo.CurrentUICulture, SR.Arg_CATypeResolutionFailed, typeName));
+                    SR.Format(CultureInfo.CurrentUICulture, SR.Arg_CATypeResolutionFailed, typeName));
```

Comment:
My guess is that `CurrentUICulture` is used by accident in these, and it should be fine to delete it and just use default as well.

Reply:
The formatting of typeNames, Types and other reflection types is not culture specific, so the explicit culture should not be needed when these items are involved.

Reply:
I'll go through and remove the cultures in such places.